packages/text/src/react/__stories__/index.story.tsx

```diff
@@ -18,27 +18,27 @@ const PaddingDecorator = storyFn => (
 
 const heading = storiesOf('Heading', module).addDecorator(PaddingDecorator)
 Object.keys(Heading.sizes).forEach(size =>
-  heading.add(`size: ${size}`, _ => (
-    <Heading size={size}>
+  heading.add(`size: ${size}`, () => (
+    <Heading size={size as keyof typeof Heading.sizes}>
       <h1>{size}</h1>
     </Heading>
   ))
 )
```

Comment:
i think this work if you do `Object.values` instead....maybe...and then you don't have to cast `size`